ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -199,6 +207,7 @@ function filter_test_by_capability() {
 function filter_tests() {
     filter_test_by_capability
     filter_test_by_fips
+    filter_test_by_proxy
```

Comment:
Shall we sort the list alphabetically?

Reply:
Generally it should. 
The point here is that some of them add `and @xxx`, and some of them add `and not @yyy`. 
So instead of list here alphanumerically here, then got `@xxx1 and not @xxx2 and @xxx3 and not @xxx4`.
I would prefer here the list is not alphanumerically, but a easier to read `@xxx1 and @xxx2 and not @xxx3 and not @xxx4`

Reply:
But I don't see you sort the options by `not` or no `not`:
```
In function: filter_test_by_proxy
E2E_RUN_TAGS: '@proxy and not @fips and @proxy and not @customer
    and not @destructive
    and not @flaky
    and not @inactive
    and not @long-duration
    and not @prod-only
    and not @qeci
    and not @security
    and not @stage-only
    and not @upgrade-check
    and not @upgrade-prepare
'
```

Reply:
Because the second `and @proxy` (coming from static filter) is what we will optimize, as a follow up of this PR.

Reply:
The [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/48044/rehearse-48044-periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-gcp-ipi-proxy-etcd-encryption-arm-f14/1750370257505095680/artifacts/gcp-ipi-proxy-etcd-encryption-arm-f14/cucushift-e2e/build-log.txt) is available now.